Updating from 15.1.1 to 15.2.0 has @angular-devkit/build-angular@">=0.1102.19 <15.0.0" error

### Current Behavior

I was just trying to `nx migrate latest` from my recent NX workspace 15.1.1.  The migrate command worked, and when trying to do a `npm install` I get the following error:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: chowarmaan@0.3.0
npm ERR! Found: @angular-devkit/build-angular@14.2.1
npm ERR! node_modules/@angular-devkit/build-angular
npm ERR!   dev @angular-devkit/build-angular@"15.0.0" from the root project
npm ERR!   peer @angular-devkit/build-angular@">=0.1102.19 <15.0.0" from jest-preset-angular@12.2.2
npm ERR!   node_modules/jest-preset-angular
npm ERR!     dev jest-preset-angular@"12.2.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! dev @angular-devkit/build-angular@"15.0.0" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: @angular/compiler-cli@15.0.0
npm ERR! node_modules/@angular/compiler-cli
npm ERR!   peer @angular/compiler-cli@"^15.0.0" from @angular-devkit/build-angular@15.0.0
npm ERR!   node_modules/@angular-devkit/build-angular
npm ERR!     dev @angular-devkit/build-angular@"15.0.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\stevens\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\stevens\AppData\Local\npm-cache\_logs\2022-11-18T16_36_29_371Z-debug-0.log
```


### Expected Behavior

npm install should properly install all packages and dependencies

### Github Repo

_No response_

### Steps to Reproduce

1. nx migrate latest on a 15.1.1 workspace
2. npm install
3. 


### Nx Report

```shell
>  NX   Report complete - copy this into the issue template

   Node : 18.5.0
   OS   : win32 x64
   npm  : 8.12.1

   nx : 15.1.1
   @nrwl/angular : 15.1.1
   @nrwl/cypress : 15.1.1
   @nrwl/detox : Not Found
   @nrwl/devkit : 15.1.1
   @nrwl/esbuild : Not Found
   @nrwl/eslint-plugin-nx : 15.1.1
   @nrwl/expo : Not Found
   @nrwl/express : Not Found
   @nrwl/jest : 15.1.1
   @nrwl/js : 15.1.1
   @nrwl/linter : 15.1.1
   @nrwl/nest : 15.1.1
   @nrwl/next : Not Found
   @nrwl/node : 15.1.1
   @nrwl/nx-cloud : Not Found
   @nrwl/nx-plugin : Not Found
   @nrwl/react : Not Found
   @nrwl/react-native : Not Found
   @nrwl/rollup : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : 15.1.1
   @nrwl/web : Not Found
   @nrwl/webpack : 15.1.1
   @nrwl/workspace : 15.1.1
   typescript : 4.8.4
   ---------------------------------------
   Local workspace plugins:
   ---------------------------------------
   Community plugins:
         @ngrx/component-store: 14.0.2
         @ngrx/effects: 14.0.2
         @ngrx/entity: 14.0.2
         @ngrx/router-store: 14.0.2
         @ngrx/store: 14.0.2
         @jscutlery/semver: 2.29.0
         @ngrx/schematics: 14.0.2
         @ngrx/store-devtools: 14.0.2

```
```


### Failure Logs

```shell
eresolve-report.txt
# npm resolution error report

2022-11-18T16:36:38.296Z

While resolving: chowarmaan@0.3.0
Found: @angular-devkit/build-angular@14.2.1
node_modules/@angular-devkit/build-angular
  dev @angular-devkit/build-angular@"15.0.0" from the root project
  peer @angular-devkit/build-angular@">=0.1102.19 <15.0.0" from jest-preset-angular@12.2.2
  node_modules/jest-preset-angular
    dev jest-preset-angular@"12.2.2" from the root project

Could not resolve dependency:
dev @angular-devkit/build-angular@"15.0.0" from the root project

Conflicting peer dependency: @angular/compiler-cli@15.0.0
node_modules/@angular/compiler-cli
  peer @angular/compiler-cli@"^15.0.0" from @angular-devkit/build-angular@15.0.0
  node_modules/@angular-devkit/build-angular
    dev @angular-devkit/build-angular@"15.0.0" from the root project

Fix the upstream dependency conflict, or retry
this command with --force, or --legacy-peer-deps
to accept an incorrect (and potentially broken) dependency resolution.

Raw JSON explanation object:

{
  "code": "ERESOLVE",
  "edge": {
    "type": "dev",
    "name": "@angular-devkit/build-angular",
    "spec": "15.0.0",
    "error": "INVALID",
    "from": {
      "location": "C:\\Users\\stevens\\Development\\Chowarmaan"
    }
  },
  "dep": {
    "name": "@angular-devkit/build-angular",
    "version": "15.0.0",
    "whileInstalling": {
      "name": "chowarmaan",
      "version": "0.3.0",
      "path": "C:\\Users\\stevens\\Development\\Chowarmaan"
    },
    "location": "node_modules/@angular-devkit/build-angular",
    "isWorkspace": false,
    "dependents": [
      {
        "type": "dev",
        "name": "@angular-devkit/build-angular",
        "spec": "15.0.0",
        "error": "INVALID",
        "from": {
          "location": "C:\\Users\\stevens\\Development\\Chowarmaan"
        }
      }
    ]
  },
  "current": {
    "name": "@angular-devkit/build-angular",
    "version": "14.2.1",
    "location": "node_modules/@angular-devkit/build-angular",
    "isWorkspace": false,
    "dependents": [
      {
        "type": "dev",
        "name": "@angular-devkit/build-angular",
        "spec": "15.0.0",
        "error": "INVALID",
        "from": {
          "location": "C:\\Users\\stevens\\Development\\Chowarmaan"
        }
      },
      {
        "type": "peer",
        "name": "@angular-devkit/build-angular",
        "spec": ">=0.1102.19 <15.0.0",
        "from": {
          "name": "jest-preset-angular",
          "version": "12.2.2",
          "location": "node_modules/jest-preset-angular",
          "isWorkspace": false,
          "dependents": [
            {
              "type": "dev",
              "name": "jest-preset-angular",
              "spec": "12.2.2",
              "from": {
                "location": "C:\\Users\\stevens\\Development\\Chowarmaan"
              }
            }
          ]
        }
      }
    ]
  },
  "peerConflict": {
    "current": {
      "name": "@angular/compiler-cli",
      "version": "14.2.0",
      "location": "node_modules/@angular/compiler-cli",
      "isWorkspace": false,
      "dependents": [
        {
          "type": "dev",
          "name": "@angular/compiler-cli",
          "spec": "15.0.0",
          "error": "INVALID",
          "from": {
            "location": "C:\\Users\\stevens\\Development\\Chowarmaan"
          }
        },
        {
          "type": "peer",
          "name": "@angular/compiler-cli",
          "spec": "^14.0.0",
          "from": {
            "name": "@angular-devkit/build-angular",
            "version": "14.2.1",
            "location": "node_modules/@angular-devkit/build-angular",
            "isWorkspace": false,
            "dependents": [
              {
                "type": "dev",
                "name": "@angular-devkit/build-angular",
                "spec": "15.0.0",
                "error": "INVALID",
                "from": {
                  "location": "C:\\Users\\stevens\\Development\\Chowarmaan"
                }
              },
              {
                "type": "peer",
                "name": "@angular-devkit/build-angular",
                "spec": ">=0.1102.19 <15.0.0",
                "from": {
                  "name": "jest-preset-angular",
                  "version": "12.2.2",
                  "location": "node_modules/jest-preset-angular",
                  "isWorkspace": false,
                  "dependents": [
                    {
                      "type": "dev",
                      "name": "jest-preset-angular",
                      "spec": "12.2.2",
                      "from": {
                        "location": "C:\\Users\\stevens\\Development\\Chowarmaan"
                      }
                    }
                  ]
                }
              }
            ]
          }
        },
        {
          "type": "peer",
          "name": "@angular/compiler-cli",
          "spec": "^14.0.0",
          "from": {
            "name": "@ngtools/webpack",
            "version": "14.2.1",
            "location": "node_modules/@ngtools/webpack",
            "isWorkspace": false,
            "dependents": [
              {
                "type": "prod",
                "name": "@ngtools/webpack",
                "spec": "14.2.1",
                "from": {
                  "name": "@angular-devkit/build-angular",
                  "version": "14.2.1",
                  "location": "node_modules/@angular-devkit/build-angular",
                  "isWorkspace": false,
                  "dependents": [
                    {
                      "type": "dev",
                      "name": "@angular-devkit/build-angular",
                      "spec": "15.0.0",
                      "error": "INVALID",
                      "from": {
                        "location": "C:\\Users\\stevens\\Development\\Chowarmaan"
                      }
                    },
                    {
                      "type": "peer",
                      "name": "@angular-devkit/build-angular",
                      "spec": ">=0.1102.19 <15.0.0",
                      "from": {
                        "name": "jest-preset-angular",
                        "version": "12.2.2",
                        "location": "node_modules/jest-preset-angular",
                        "isWorkspace": false,
                        "dependents": [
                          {
                            "type": "dev",
                            "name": "jest-preset-angular",
                            "spec": "12.2.2",
                            "from": {
                              "location": "C:\\Users\\stevens\\Development\\Chowarmaan"
                            }
                          }
                        ]
                      }
                    }
                  ]
                }
              }
            ]
          }
        },
        {
          "type": "peer",
          "name": "@angular/compiler-cli",
          "spec": ">=11.2.14 <15.0.0",
          "from": {
            "name": "jest-preset-angular",
            "version": "12.2.2",
            "location": "node_modules/jest-preset-angular",
            "isWorkspace": false,
            "dependents": [
              {
                "type": "dev",
                "name": "jest-preset-angular",
                "spec": "12.2.2",
                "from": {
                  "location": "C:\\Users\\stevens\\Development\\Chowarmaan"
                }
              }
            ]
          }
        }
      ]
    },
    "peer": {
      "name": "@angular/compiler-cli",
      "version": "15.0.0",
      "whileInstalling": {
        "name": "chowarmaan",
        "version": "0.3.0",
        "path": "C:\\Users\\stevens\\Development\\Chowarmaan"
      },
      "location": "node_modules/@angular/compiler-cli",
      "isWorkspace": false,
      "dependents": [
        {
          "type": "peer",
          "name": "@angular/compiler-cli",
          "spec": "^15.0.0",
          "from": {
            "name": "@angular-devkit/build-angular",
            "version": "15.0.0",
            "whileInstalling": {
              "name": "chowarmaan",
              "version": "0.3.0",
              "path": "C:\\Users\\stevens\\Development\\Chowarmaan"
            },
            "location": "node_modules/@angular-devkit/build-angular",
            "isWorkspace": false,
            "dependents": [
              {
                "type": "dev",
                "name": "@angular-devkit/build-angular",
                "spec": "15.0.0",
                "error": "INVALID",
                "from": {
                  "location": "C:\\Users\\stevens\\Development\\Chowarmaan"
                }
              }
            ]
          }
        }
      ]
    }
  },
  "strictPeerDeps": false,
  "force": false,
  "isMine": true
}


npm-debug.log
0 verbose cli C:\Program Files\nodejs\node.exe C:\Program Files\nodejs\node_modules\npm\bin\npm-cli.js
1 info using npm@8.12.1
2 info using node@v18.5.0
3 timing npm:load:whichnode Completed in 1ms
4 timing config:load:defaults Completed in 2ms
5 timing config:load:file:C:\Program Files\nodejs\node_modules\npm\npmrc Completed in 3ms
6 timing config:load:builtin Completed in 3ms
7 timing config:load:cli Completed in 1ms
8 timing config:load:env Completed in 1ms
9 timing config:load:file:C:\Development\.npmrc Completed in 0ms
10 timing config:load:project Completed in 5ms
11 timing config:load:file:C:\Users\stevens\.npmrc Completed in 1ms
12 timing config:load:user Completed in 1ms
13 timing config:load:file:C:\Users\stevens\AppData\Roaming\npm\etc\npmrc Completed in 0ms
14 timing config:load:global Completed in 0ms
15 timing config:load:validate Completed in 0ms
16 timing config:load:credentials Completed in 0ms
17 timing config:load:setEnvs Completed in 1ms
18 timing config:load Completed in 15ms
19 timing npm:load:configload Completed in 15ms
20 timing npm:load:mkdirpcache Completed in 1ms
21 timing npm:load:mkdirplogs Completed in 1ms
22 verbose title npm install
23 verbose argv "install"
24 timing npm:load:setTitle Completed in 1ms
25 timing config:load:flatten Completed in 3ms
26 timing npm:load:display Completed in 4ms
27 verbose logfile logs-max:10 dir:C:\Users\stevens\AppData\Local\npm-cache\_logs
28 verbose logfile C:\Users\stevens\AppData\Local\npm-cache\_logs\2022-11-18T16_36_29_371Z-debug-0.log
29 timing npm:load:logFile Completed in 6ms
30 timing npm:load:timers Completed in 0ms
31 timing npm:load:configScope Completed in 0ms
32 timing npm:load Completed in 29ms
33 timing arborist:ctor Completed in 0ms
34 silly logfile start cleaning logs, removing 2 files
35 silly logfile done cleaning log files
36 timing idealTree:init Completed in 1339ms
37 timing idealTree:userRequests Completed in 0ms
38 silly idealTree buildDeps
39 silly fetch manifest @angular/animations@15.0.0
40 timing arborist:ctor Completed in 1ms
41 http fetch GET 200 https://registry.npmjs.org/@angular%2fanimations 1397ms (cache revalidated)
42 silly fetch manifest @angular/core@15.0.0
43 http fetch GET 200 https://registry.npmjs.org/@angular%2fcore 13ms (cache hit)
44 silly fetch manifest rxjs@7.5.6
45 http fetch GET 200 https://registry.npmjs.org/rxjs 60ms (cache revalidated)
46 silly fetch manifest zone.js@~0.11.4
47 http fetch GET 200 https://registry.npmjs.org/zone.js 75ms (cache revalidated)
48 silly fetch manifest @angular/cdk@15.0.0
49 http fetch GET 200 https://registry.npmjs.org/@angular%2fcdk 16ms (cache hit)
50 silly fetch manifest @angular/common@15.0.0
51 http fetch GET 200 https://registry.npmjs.org/@angular%2fcommon 65ms (cache revalidated)
52 silly fetch manifest @angular/compiler@15.0.0
53 http fetch GET 200 https://registry.npmjs.org/@angular%2fcompiler 156ms (cache revalidated)
54 silly fetch manifest @angular/forms@15.0.0
55 http fetch GET 200 https://registry.npmjs.org/@angular%2fforms 111ms (cache revalidated)
56 silly fetch manifest @angular/platform-browser@15.0.0
57 http fetch GET 200 https://registry.npmjs.org/@angular%2fplatform-browser 75ms (cache revalidated)
58 silly fetch manifest @angular/material@15.0.0
59 http fetch GET 200 https://registry.npmjs.org/@angular%2fmaterial 10ms (cache hit)
60 silly fetch manifest @angular/platform-browser-dynamic@15.0.0
61 http fetch GET 200 https://registry.npmjs.org/@angular%2fplatform-browser-dynamic 129ms (cache revalidated)
62 silly fetch manifest @angular/router@15.0.0
63 http fetch GET 200 https://registry.npmjs.org/@angular%2frouter 101ms (cache revalidated)
64 silly fetch manifest @angular-devkit/build-angular@15.0.0
65 http fetch GET 200 https://registry.npmjs.org/@angular-devkit%2fbuild-angular 206ms (cache revalidated)
66 silly fetch manifest @angular/compiler-cli@15.0.0
67 http fetch GET 200 https://registry.npmjs.org/@angular%2fcompiler-cli 102ms (cache revalidated)
68 silly fetch manifest typescript@4.8.4
69 http fetch GET 200 https://registry.npmjs.org/typescript 54ms (cache revalidated)
70 silly fetch manifest @angular/localize@^15.0.0
71 http fetch GET 200 https://registry.npmjs.org/@angular%2flocalize 64ms (cache revalidated)
72 silly fetch manifest @angular/platform-server@^15.0.0
73 http fetch GET 200 https://registry.npmjs.org/@angular%2fplatform-server 93ms (cache updated)
74 silly fetch manifest @angular/service-worker@^15.0.0
75 http fetch GET 200 https://registry.npmjs.org/@angular%2fservice-worker 71ms (cache revalidated)
76 silly fetch manifest karma@^6.3.0
77 http fetch GET 200 https://registry.npmjs.org/karma 133ms (cache revalidated)
78 silly fetch manifest ng-packagr@^15.0.0
79 http fetch GET 200 https://registry.npmjs.org/ng-packagr 54ms (cache revalidated)
80 silly fetch manifest tailwindcss@^2.0.0 || ^3.0.0
81 http fetch GET 200 https://registry.npmjs.org/tailwindcss 76ms (cache revalidated)
82 silly fetch manifest tslib@^2.3.0
83 http fetch GET 200 https://registry.npmjs.org/tslib 83ms (cache revalidated)
84 silly fetch manifest protractor@^7.0.0
85 http fetch GET 200 https://registry.npmjs.org/protractor 163ms (cache revalidated)
86 silly fetch manifest @angular/language-service@15.0.0
87 http fetch GET 200 https://registry.npmjs.org/@angular%2flanguage-service 93ms (cache revalidated)
88 silly fetch manifest @nrwl/angular@15.2.0
89 http fetch GET 200 https://registry.npmjs.org/@nrwl%2fangular 204ms (cache revalidated)
90 silly fetch manifest @nrwl/cli@15.2.0
91 http fetch GET 200 https://registry.npmjs.org/@nrwl%2fcli 533ms (cache revalidated)
92 silly fetch manifest @nrwl/cypress@15.2.0
93 http fetch GET 200 https://registry.npmjs.org/@nrwl%2fcypress 192ms (cache revalidated)
94 silly fetch manifest cypress@^11.0.0
95 http fetch GET 200 https://registry.npmjs.org/cypress 100ms (cache revalidated)
96 silly fetch manifest @nrwl/devkit@15.2.0
97 http fetch GET 200 https://registry.npmjs.org/@nrwl%2fdevkit 141ms (cache revalidated)
98 silly fetch manifest nx@15.2.0
99 http fetch GET 200 https://registry.npmjs.org/nx 71ms (cache revalidated)
100 silly fetch manifest @swc-node/register@^1.4.2
101 http fetch GET 200 https://registry.npmjs.org/@swc-node%2fregister 144ms (cache revalidated)
102 silly fetch manifest @swc/core@>= 1.3
103 http fetch GET 200 https://registry.npmjs.org/@swc%2fcore 141ms (cache revalidated)
104 silly fetch manifest @nrwl/eslint-plugin-nx@15.2.0
105 http fetch GET 200 https://registry.npmjs.org/@nrwl%2feslint-plugin-nx 499ms (cache revalidated)
106 silly fetch manifest @typescript-eslint/parser@5.36.1
107 http fetch GET 200 https://registry.npmjs.org/@typescript-eslint%2fparser 121ms (cache revalidated)
108 silly fetch manifest eslint@~8.15.0
109 http fetch GET 200 https://registry.npmjs.org/eslint 73ms (cache revalidated)
110 silly fetch manifest eslint-config-prettier@8.1.0
111 http fetch GET 200 https://registry.npmjs.org/eslint-config-prettier 42ms (cache revalidated)
112 silly fetch manifest @nrwl/jest@15.2.0
113 http fetch GET 200 https://registry.npmjs.org/@nrwl%2fjest 107ms (cache revalidated)
114 silly fetch manifest @nrwl/linter@15.2.0
115 http fetch GET 200 https://registry.npmjs.org/@nrwl%2flinter 156ms (cache revalidated)
116 silly fetch manifest @nrwl/nest@15.2.0
117 http fetch GET 200 https://registry.npmjs.org/@nrwl%2fnest 533ms (cache revalidated)
118 silly fetch manifest @nrwl/node@15.2.0
119 http fetch GET 200 https://registry.npmjs.org/@nrwl%2fnode 128ms (cache revalidated)
120 silly fetch manifest @nrwl/workspace@15.2.0
121 http fetch GET 200 https://registry.npmjs.org/@nrwl%2fworkspace 135ms (cache revalidated)
122 silly fetch manifest prettier@^2.6.2
123 http fetch GET 200 https://registry.npmjs.org/prettier 60ms (cache revalidated)
124 timing idealTree Completed in 8781ms
125 timing command:install Completed in 8795ms
126 verbose stack Error: could not resolve
126 verbose stack     at PlaceDep.failPeerConflict (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\place-dep.js:549:25)
126 verbose stack     at PlaceDep.place (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\place-dep.js:199:21)
126 verbose stack     at new PlaceDep (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\place-dep.js:73:10)
126 verbose stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\build-ideal-tree.js:990:31
126 verbose stack     at Array.map (<anonymous>)
126 verbose stack     at [buildDepStep] (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\build-ideal-tree.js:990:8)
126 verbose stack     at async Arborist.buildIdealTree (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\build-ideal-tree.js:218:7)
126 verbose stack     at async Promise.all (index 1)
126 verbose stack     at async Arborist.reify (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\reify.js:153:5)
126 verbose stack     at async Install.exec (C:\Program Files\nodejs\node_modules\npm\lib\commands\install.js:156:5)
127 verbose cwd C:\Development
128 verbose Windows_NT 10.0.19042
129 verbose node v18.5.0
130 verbose npm  v8.12.1
131 error code ERESOLVE
132 error ERESOLVE could not resolve
133 error
134 error While resolving: [1mchowarmaan[22m@[1m0.3.0[22m
134 error Found: [1m@angular-devkit/build-angular[22m@[1m14.2.1[22m[2m[22m
134 error [2mnode_modules/@angular-devkit/build-angular[22m
134 error   [33mdev[39m [1m@angular-devkit/build-angular[22m@"[1m15.0.0[22m" from the root project
134 error   [35mpeer[39m [1m@angular-devkit/build-angular[22m@"[1m>=0.1102.19 <15.0.0[22m" from [1mjest-preset-angular[22m@[1m12.2.2[22m[2m[22m
134 error   [2mnode_modules/jest-preset-angular[22m
134 error     [33mdev[39m [1mjest-preset-angular[22m@"[1m12.2.2[22m" from the root project
134 error
134 error Could not resolve dependency:
134 error [33mdev[39m [1m@angular-devkit/build-angular[22m@"[1m15.0.0[22m" from the root project
134 error
134 error Conflicting peer dependency: [1m@angular/compiler-cli[22m@[1m15.0.0[22m[2m[22m
134 error [2mnode_modules/@angular/compiler-cli[22m
134 error   [35mpeer[39m [1m@angular/compiler-cli[22m@"[1m^15.0.0[22m" from [1m@angular-devkit/build-angular[22m@[1m15.0.0[22m[2m[22m
134 error   [2mnode_modules/@angular-devkit/build-angular[22m
134 error     [33mdev[39m [1m@angular-devkit/build-angular[22m@"[1m15.0.0[22m" from the root project
134 error
134 error Fix the upstream dependency conflict, or retry
134 error this command with --force, or --legacy-peer-deps
134 error to accept an incorrect (and potentially broken) dependency resolution.
134 error
134 error See C:\Users\stevens\AppData\Local\npm-cache\eresolve-report.txt for a full report.
135 verbose exit 1
136 timing npm Completed in 8938ms
137 verbose unfinished npm timer reify 1668789389498
138 verbose unfinished npm timer reify:loadTrees 1668789389508
139 verbose unfinished npm timer idealTree:buildDeps 1668789390849
140 verbose unfinished npm timer idealTree:#root 1668789390849
141 verbose code 1
142 error A complete log of this run can be found in:
142 error     C:\Users\stevens\AppData\Local\npm-cache\_logs\2022-11-18T16_36_29_371Z-debug-0.log
```


### Additional Information

_No response_

--- a/packages/angular/migrations.json
+++ b/packages/angular/migrations.json
@@ -1665,6 +1665,10 @@
         "@angular-eslint/template-parser": {
           "version": "~15.0.0",
           "alwaysAddToPackageJson": false
+        },
+        "jest-preset-angular": {
+          "version": "12.2.3",
+          "alwaysAddToPackageJson": false
         }
       }
     }

--- a/packages/angular/src/utils/versions.ts
+++ b/packages/angular/src/utils/versions.ts
@@ -23,7 +23,7 @@ export const postcssUrlVersion = '~10.1.3';
 export const autoprefixerVersion = '^10.4.0';
 export const tsNodeVersion = '10.9.1';
 
-export const jestPresetAngularVersion = '~12.2.2';
+export const jestPresetAngularVersion = '~12.2.3';
 export const protractorVersion = '~7.0.0';
 export const karmaVersion = '~6.4.0';
 export const karmaChromeLauncherVersion = '~3.1.0';

